src/class/template.php

```diff
@@ -782,6 +787,12 @@ function lovd_changeURL ()
               '</TABLE>' . "\n\n" .
               '<IMG src="gfx/trans.png" alt="" width="' . $nTotalTabWidth . '" height="0">' . "\n\n");
 
+        // Measure the height of the sticky header (can depend on announcements or
+        // font settings and such), and adapt the menu table to have a margin of this height.
+        print('<SCRIPT type="text/javascript">' . "\n" .
+              '  $("table.logo :eq(1)").css("margin-top", $("#stickyheader").height() + "px");' . "\n" .
```

Comment:
Maybe it's best to use outerHeight() instead of height(), that includes padding and border (if that's ever added to the header div).